### PR TITLE
Update publish an extension documentation

### PR DESCRIPTION
**Pull request by gebansal**

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 1]
```diff
@@ -1,297 +1,188 @@
...
-The description for your users of what your Extension does.  
+##### YouTube video URL requirements 
 
-**Possible Values**:  
+Ensure your video meets the following requirements: 
 
-*   {plain_text}: Less than 10,000 characters.  
+*   Verify that the content of YouTube video complies with the [Microsoft Edge Addons Catalog Developer Policies document][MicrosoftEdgeAddonsCatalogDeveloperPolicies]. 
+*   Turn off advertisements on your video. For more information, see [Set your default ad formats][GoogleYoutubeAnswer2531367Topic7072227] and [Ads on embedded videos][GoogleYoutubeAnswer132596] pages.  
+*   Turn on embedding for your videos.  For more information, see [Embed videos \& playlists][GoogleYoutubeAnswer171780] page.
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:123]:
```suggestion
*   Turn off advertisements on your video. For more information, see [Set your default ad formats][GoogleYoutubeAnswer2531367Topic7072227] and [Ads on embedded videos][GoogleYoutubeAnswer132596].  
*   Turn on embedding for your videos.  For more information, see [Embed videos \& playlists][GoogleYoutubeAnswer171780].
```

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 2]
```diff
@@ -1,297 +1,188 @@
...
-The Submission options page of the Extension submission process is where you provide more information to help us test your product properly.  This is an optional step, but it is recommended for many submissions.  
-
-### Publishing hold options  
-
-By default, your submission is published as soon as it passes certification.  You may choose to place a hold on publishing your submission until a specific date.  The options in this section are described below.  
-
-*   **Publish your submission as soon as it passes certification**  
-
-    This is the default selection.  It means that your submission begins the publishing process as soon as it passes certification  
+> If you are facing any issues in the submission or registration process, you can file a support ticket [here][ExtensionsSupportForm] or write to us at *ext_dev_support@microsoft.com*
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:163]:
```suggestion
> If you are facing any issues in the submission or registration process, you can file a support ticket [here][ExtensionsSupportForm] or write to us at ext_dev_support@microsoft.com.
```

**Review by Reezaali — CHANGES_REQUESTED** [review]
Please review the feedback. Thanks.

**Review by Reezaali — APPROVED** [review]

**Comment by gebansal** [comment]
This documentation is ready to be published.

**Review by jm-trd-ms — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**